File: plugin/testDataSwift/shared/basics/Casting.shared.swift
//Package: com.test
//Converted using Khrysalis2

import Foundation



public func main() -> Void {
    var value: Any?  = nil
    var asString = value as? String
    value = "XP"
    var forced = value as! String
    print(value ?? "Nope")
    print(forced)
}
 

File: plugin/testDataOutputTs/shared/basics/Casting.shared.ts
//Package: com.test
//Converted using Khrysalis2


export function main(): void {
    let value : any | null = null;
    const asString = ((): string | null => { const _item: any = value; if(typeof _item == "string") return _item; else return null })();
    value = "XP";
    const forced = value as string;
    console.log(value ?? "Nope");
    console.log(forced)
}
 
